### ISSUE TITLE
Scroll-fixing of background page doesn't work in some cases

### ISSUE THREAD

**Issue by jasomdotnet** [englishextra/iframe-lightbox#18]
[See](https://privnote.com/O4FcGGvp#V87mKDEGY) this page. Background page [scroll fix](https://stackoverflow.com/questions/4770025/how-to-disable-scrolling-temporarily) does not work there.

**Comment by englishextra** [comment]
that;'s because

```css
body.iframe-lightbox--open {
	overflow: hidden;
}
```
the dev should do it on their own in their overriding style
```css
body.iframe-lightbox--open {
	overflow: hidden !important;
}
```

Also I see that the HTML element has

```css
html {overflow-y:scroll;}
```
will be soon
```css
html.iframe-lightbox--open,
body.iframe-lightbox--open {
	overflow: hidden !important;
}
```

**Comment by englishextra** [comment]
Fixed in v0.2.8

**Comment by jasomdotnet** [comment]
excellent
